packages/eslint-config/eslint-config.json

```diff
@@ -52,6 +52,8 @@
         ]
       }
     ],
+    // Work around https://github.com/import-js/eslint-plugin-import/issues/1810
+    "import/no-unresolved": ["error", { "ignore": ["ava"] }],
```

Comment:
I think I'm ok not understanding this. But if you think it needs my attention, please let me know.